app/uk/gov/nationalarchives/omega/editorial/services/jms/StubServer.scala

```diff
@@ -81,8 +82,25 @@ class StubServer @Inject() (responseBuilder: ResponseBuilder) {
       requestMessageId <- responseBuilder.jmsMessageId(jmsMessage)
       _                <- logger.info(s"got a message with ID $requestMessageId")
       responseText     <- responseBuilder.createResponseText(jmsMessage)
-      responseMessage  <- messageFactory.makeTextMessage(responseText)
-      _ = responseMessage.setJMSCorrelationId(requestMessageId)
-    } yield responseMessage
+      replyMessage     <- messageFactory.makeTextMessage(responseText)
+      _ = replyMessage.setJMSCorrelationId(requestMessageId)
+      _ = replyMessage.setStringProperty(MessageProperties.OMGApplicationID, "PACS001")
+      _ = replyMessage.setStringProperty(
+            MessageProperties.OMGMessageTypeID,
+            getReplyMessageType(jmsMessage.getStringProperty(MessageProperties.OMGMessageTypeID))
+          )
+      _ = replyMessage.setStringProperty(MessageProperties.OMGMessageFormat, "application/json")
+      _ = replyMessage.setStringProperty(MessageProperties.OMGReplyAddress, "PACS001.request")
```

Comment:
Is the reply queue `omega-editorial-web-application-instance-1`

Reply:
No. In the API schema @adamretter has specified that all messages must have an `OMGReplyAddress` property set which contains the queue name for message replies, and that matches [a regex shown here](https://github.com/nationalarchives/ctd-omega-services/blob/734fef1387aa4145a27636ea7fdafbc3ae76fe9b/schema/omega-services.async.api.yaml#L380). This regex requires a dot separator but it seems that neither SQS nor ElasticMQ support dots in the queue name. We will need to discuss this further with Adam.

Reply:
OK